pynetbox_data_uploader/lib/user_methods/csv_to_netbox.py

```diff
@@ -24,63 +27,75 @@ def __init__(self, url: str, token: str):
         :param token: The Netbox auth token.
         """
         self.netbox = NetboxConnect(url, token).api_object()
-        self.format_dict = FormatDict(self.netbox)
         self.exist = NetboxCheck(self.netbox)
         self.create = NetboxCreate(self.netbox)
+        self.query_dataclass = QueryDataclass(self.netbox)
 
-    def read_csv(self, file_path) -> List:
+    @staticmethod
+    def check_file_path(file_path: str):
+        """
+        This method checks if the filepath is valid. Raises an exception if it's invalid.
+        :param file_path: The path to the csv file.
+        """
+        print("Checking filepath...")
+        valid = Path(file_path).exists()
+        if not valid:
+            raise FileNotFoundError(f"Filepath: {file_path} is not valid.")
+        print("Filepath valid.")

```

Comment:
Just wondering if this should return true if the filepath is valid?

Reply:
It only matters if the file path is invalid so we throw an error. If the file path is valid the code will continue.